resources/assets/js/app/views/topics/topicCreate.js

```diff
@@ -49,7 +49,7 @@ module.exports = Marionette.LayoutView.extend({
             e.preventDefault();
             this.model.save({}, {
                 success: function (model, response) {
-                    Backbone.history.navigate('topics/' + model.get('id'), {trigger: true});
+                    Backbone.history.navigate('topics/' + model.get('slug'), {trigger: true});
```

Comment:
I think there is need some condition:
if slug is presented then navigate to link with slug, otherwise link with id